### PR TITLE
CTV-227 Content changes

### PR DESCRIPTION
**Pull request by AmritSidhu**

### CODE REVIEW

File: spec/features/user_visits_problem_form_spec.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'The problem form' do
+
+  it 'should visit problem form from when problem is selected' do
```
Comment (Rosa-Fox) [inline spec/features/user_visits_problem_form_spec.rb]: really small thing but maybe remove the word 'from'?

File: spec/features/user_visits_problem_form_spec.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    expect(page).to have_content "Message can't be blank"
+    expect(page).to have_content "Your name can't be blank"
+    expect(find_field(id: 'support_service_name')).to match_selector('.form-control-error')
+    expect(find_field(id: 'support_your_email_address')).to match_selector('.form-control-error')
+  end
+
+  it 'should maintain the input values after validation fails' do
+    visit '/support/problem'
+    fill_in('Message', with: 'Example message')
+    fill_in('Your name', with: '') #maintain nput values when another field is missing content
```
Comment (Rosa-Fox) [inline spec/features/user_visits_problem_form_spec.rb]: missing an `i` in input in the comment

**Review by Rosa-Fox — COMMENTED** [review]
Hi @AmritSidhu,

Great work 👍 I added some tiny comments to change spelling/wording but code looks good.

File: spec/features/user_visits_problem_form_spec.rb (hunk 1 above)
Comment (AmritSidhu) [reply spec/features/user_visits_problem_form_spec.rb]: Done

File: spec/features/user_visits_problem_form_spec.rb (hunk 2 above)
Comment (AmritSidhu) [reply spec/features/user_visits_problem_form_spec.rb]: Done

**Comment by Rosa-Fox** [comment]
LGTM :)

**Pull request merged**